src/vsphere_cpi/spec/integration/cpi_advanced_config_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rspec'
+require 'integration/spec_helper'
+
+describe '#add advanced cpi config' do
+  subject(:cpi) do
+    VSphereCloud::Cloud.new(cpi_options('default_disk_type' => 'thin'))
+  end
+
+  context 'when it creates a stemcell on vsphere' do
+    it 'add an advanced cpi config config.SDDC.cpi to vpxd.cfg file' do
+      begin
+        stemcell_id = upload_stemcell(cpi)
+        cpi_advanced_prop_key='config.SDDC.cpi'
+        expect(cpi.client.service_content.setting.
```

Comment:
extract the value in a variable, so that its more readable
expect(advanced_cpi_config).to eql('true)